tsdb/track.json

```diff
@@ -0,0 +1,32 @@
+{% import "rally.helpers" as rally with context %}
+
+{
+  "version": 2,
+  "description": "metricbeat information for elastic-app k8s cluster",
+  "indices": [
+    {
+      "name": "tsdb",
+      "body": "index.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "tsdb",
+      "#base-url": "https://rally-tracks.elastic.co/tsdb",
```

Comment:
I've commented this out because I'd like a review on the tools first and then I'll upload the anonymized file.